res/fingerprint-server-api.yaml

```diff
@@ -291,6 +314,7 @@ paths:
         how long you should wait before making a follow-up request. The value is
         non-negative decimal integer indicating the seconds to delay after the
         response is received.
+      x-flattern-filter: true
```

Comment:
Isn't there a typo? Did you mean `x-flatten-filter` as in make the filter flat?

Reply:
It is and I spread it everywhere in this PR. Thank you, fixed!